bitrise.yml

```diff
@@ -240,6 +240,48 @@ workflows:
     - path::./:
         title: Step test
 
+  test_support_library:
+    envs:
+    - SAMPLE_APP_URL: https://github.com/godrei/JetPackTest.git
```

Comment:
Samples should be located in bitrise-io organization.
BTW all other sample URLs in this file points to bitrise-samples org which is deprecated and contains no repos now (there are redirects to bitrise-io).

Reply:
Fixed.